cns/restserver/ipam_test.go

```diff
@@ -575,6 +579,93 @@ func TestIPAMMarkIPCountAsPending(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected failure releasing IP: %+v", err)
 	}
+
+	// Try to release IP when no IP can be released. It should return error and ips will be nil
+	ips, err = svc.MarkIPAsPendingRelease(1)
+	if err == nil || ips != nil {
+		t.Fatalf("Unexpected failure releasing IP: %+v", err)
```

Comment:
I think the message should be different. If the error was nil, we should print that an error was expected. If there were ips returned, we should print that 0 ips were expected